### ISSUE TITLE
Evaluate project future due to hetzner cloud load balancers

### ISSUE THREAD

**Issue by cbeneke** [cbeneke/hcloud-fip-controller#51]
Hetzner Cloud just released load balancer as a service practically rendering this controller obsolete. It should at least be documented, that load balancer as a service should be preferred (as it's a native kubernetes object and should be included in the hcloud cloud controller).

**Comment by phiphi282** [comment]
As a matter of fact, the load balancer is already included in the hcloud-cloud-controller as of commit https://github.com/hetznercloud/hcloud-cloud-controller-manager/commit/df90212c1c022a1cb892c19466d6efa38cb925f2.

We should definitely add a note at the top of the README.md. 
I can't think of a usecase currently, where I would use the fip-controller instead of the load-balancer, but I would still leave this issue open for a while.

**Comment by drallgood** [comment]
If you still want to use your own LB, you still need this. Or any other thing that requires a floating IP

**Comment by cbeneke** [comment]
Hey @drallgood I'll not take down the project, but since hetzner released a solution to what this project is a workaround for I would strongly suggest to look into using hetzner loadbalancers (as of my knowledge now).
I haven't found the time to actually test the LB implementation from hetzner, but knowing their project I assume the solution will be both more stable and HA than a workaround can ever be :)

**Comment by drallgood** [comment]
I've tested it. 
Given the current limitations and proposed pricing, I've decided that my current solution is much better than what Hetzner is offering

**Comment by spatialy** [comment]
Hi, @drallgood can you share your experience testing the LB from Hetzner?

Our current solution is based on this project and MetalLB, but as @cbeneke state, we are looking into the new LB.

Best

**Comment by drallgood** [comment]
@spatialy I tested it for a few hours at most, so take everything I'm about to say with a grain of salt.
it works. Feedback is minimal so it was a bit of a challenge to figure out why my services were not showing up as healthy at first.
But other than that, it was easy enough to set up and seemed to work fine.

My issue is, that I need a larger amount of ports (for web, email, and GitLab for the most part) and the current beta only offers 5 ports (or "services" as they call it).
Given that the proposed pricing I found on the website for 15 services would be close to EUR20/month, I can just roll my own HA proxy, or stick with MetalLB.
That's not only more flexible, but also cheaper.
